### DISCUSSION TITLE
Incorporating mkcert as a certificate manager for Hetty.

### DISCUSSION THREAD

**Discussion by Lawri-van-Buel** [dstotijn/hetty#38] [Ideas] [answered]
For the start of this see [Pull request 21](https://github.com/dstotijn/hetty/pull/21#issuecomment-705702540)

Basically the idea is to use [Mkcert](https://github.com/FiloSottile/mkcert)  as a library for managing the root CA and it’s installation, freeing up Hetty to do the proxy bits (keeping in line with the philosophy of doing 1 thing but doing it really well.

We can even offload some of the current parts dealing with x509 generation to the library. 

I would love to get other peoples insides into this idea before anyone starts working on it.

**Comment by dstotijn** [comment, marked as answer]
Thanks for the feedback! This has been added in `v0.6`. See [this guide](https://hetty.xyz/docs/guides/trust-root-ca/) for details.
